### PR TITLE
[EDU-4913] [EDU-4912] feature: add query connected users data guide

### PR DESCRIPTION
**Pull request by MarianaAguilera**
### Related issue:

[EDU-4913] [EDU-4912]

### Changes

- Add query connected users data guide, EN/PT.
- Add the link to the Guides page.
- Add information about the new dataset to existing GraphQL documentation.

### Additional links

n/a.

[EDU-4913]: https://aziontech.atlassian.net/browse/EDU-4913?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[EDU-4912]: https://aziontech.atlassian.net/browse/EDU-4912?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by gabriel-alves-azion — APPROVED** [review]

File: src/content/docs/en/pages/devtools/api-graphql/overview/overview.mdx [hunk 1]
```diff
@@ -35,6 +35,7 @@ With the Real-Time Metrics GraphQL API, you can, for example:
 - Get IPs with most requests.
 - See domains with least requests.
 - Analyze quantity of requests by HTTP methods.
+- Consult the number of users currently connected to your live streamings.
```
Comment (brunomrpx) [inline src/content/docs/en/pages/devtools/api-graphql/overview/overview.mdx:38]: suggestion: Mesma observação sobre o `currently` aqui.

File: src/content/docs/en/pages/guides/graphql/connected-users-dataset.mdx [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+namespace: docs_guides_query_connected_users_graphql
+permalink: /documentation/products/guides/query-connected-users-data-with-graphql/
+menu_namespace: graphqlMenu
+---
+
+The **connectedUsers** dataset lets you get real-time aggregated data, provided by Azion Live Ingest, related to the number of users currently connected to your applications with live streamings. This dataset is part of the Real-Time Metrics GraphQL API and is generated from the raw HTTP events, which are populated just for clients with live streaming configurations.
+
+This information can be accessed through the GraphQL API, allowing you to transfer this data to a third-party platform, and enabling you to further analysis and review of user activity and engagement. Additionally, the data is available for up to *2 years*.
+
+This guide will explain how to query Connected Used data using the GraphiQL playground.
```
Comment (brunomrpx) [inline src/content/docs/en/pages/guides/graphql/connected-users-dataset.mdx:14]:
```suggestion
This guide will explain how to query Connected Users data using the GraphiQL playground.
```

File: src/content/docs/en/pages/guides/graphql/connected-users-dataset.mdx [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+
+Where:
+
+| Field | Description |
+|----------|----------|
+| `limit` | Specifies the maximum number of results to return. System maximum: `10000` |
+| `filter` | Defines the criteria used to filter the data returned by the query |
+| `tsRange` | A subfield of `filter`. Specifies a time range for filtering data. It includes `begin` and `end` fields to define the start and end timestamps. Format: `"YYYY-MM-DDTHH:mm:ss"`; example: `"2024-04-11T00:00:00"` |
+| `groupBy` | Specifies the fields by which the query results should be grouped. Example: `[ts]` |
+| `orderBy` | Specifies the order in which the results should be returned. Accepted values: `[ts_DESC]`, for descending order, and `[ts_ASC]`, for ascending order |
```
Comment (brunomrpx) [inline src/content/docs/en/pages/guides/graphql/connected-users-dataset.mdx:58]: suggestion: Talvez algo assim pois os valores aceitos podem ser vários.
```suggestion
| `orderBy` | Specifies the order in which the results should be returned. Examples: `[ts_DESC]`, for descending order, and `[ts_ASC]`, for ascending order |
```

**Review by brunomrpx — APPROVED** [review]
LGTM, deixei apenas algumas sugestões, mas de resto parece ótimo! 🚀

File: src/content/docs/en/pages/devtools/api-graphql/features/features.mdx [hunk 4]
```diff
@@ -41,12 +41,13 @@ Four datasets are deprecated and were replaced by other datsets:
...
-| tieredCacheMetrics | Request events registered by [Tiered Cache](/en/documentation/products/build/edge-application/tiered-cache/). |
+| dataStreamedMetrics | Sent events of data by [Data Stream](/en/documentation/products/observe/data-stream/) to the clients' endpoint |
+| edgeFunctionsMetrics | Events executed by [Edge Functions](/en/documentation/products/build/edge-application/edge-functions/) |
+| httpMetrics | Request events registered by [Edge Application](/en/documentation/products/build/edge-application/) and [Edge Firewall](/en/documentation/products/secure/edge-firewall/) |
+| edgeDnsQueriesMetrics | Query events performed on [Edge DNS](/en/documentation/products/secure/edge-dns/) |
+| imagesProcessedMetrics | Image processing events by [Image Processor](/en/documentation/products/build/edge-application/image-processor/) |
+| tieredCacheMetrics | Request events registered by [Tiered Cache](/en/documentation/products/build/edge-application/tiered-cache/) |
+| connectedUsersMetrics | Query the number of users currently connected to your applications, using [Live Ingest](/en/documentation/products/guides/query-connected-users-data-with-graphql/) |
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/api-graphql/features/features.mdx:50]:
```suggestion
| connectedUsersMetrics | Query the number of users connected to your applications, using [Live Ingest](/en/documentation/products/guides/query-connected-users-data-with-graphql/) |
```

File: src/content/docs/pt-br/pages/devtools/api-graphql/recursos/recursos-gql.mdx [hunk 5]
```diff
@@ -39,12 +39,13 @@ Quatro conjuntos de dados foram descontinuados e substituiídos por outros:
...
+| dataStreamedMetrics | Registros de envio de dados do [Data Stream](/pt-br/documentacao/produtos/observe/data-stream/) para o endpoint do cliente |
+| edgeFunctionsMetrics | Eventos de execução do [Edge Functions](/pt-br/documentacao/produtos/build/edge-application/edge-functions/) |
+| httpMetrics | Eventos de requisições registradas pelo [Edge Application](/pt-br/documentacao/produtos/build/edge-application/) e [Edge Firewall](/pt-br/documentacao/produtos/secure/edge-firewall/) |
+| edgeDnsQueriesMetrics | Eventos de consultas realizadas no [Edge DNS](/pt-br/documentacao/produtos/secure/edge-dns/) |
+| imagesProcessedMetrics |  Eventos de processamento de imagens do [Image Processor](/pt-br/documentacao/produtos/build/edge-application/image-processor/) |
+| tieredCacheMetrics | Eventos de requisições registradas pelo [Tiered Cache](/pt-br/documentacao/produtos/build/edge-application/tiered-cache/) |
+| connectedUsersMetrics | Query the number of users currently connected to your applications, using [Live Ingest](/pt-br/documentacao/produtos/guias/query-dados-connected-users-com-graphql/) |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/api-graphql/recursos/recursos-gql.mdx:48]:
```suggestion
| connectedUsersMetrics | Query the number of users connected to your applications, using [Live Ingest](/pt-br/documentacao/produtos/guias/query-dados-connected-users-com-graphql/) |
```

File: src/content/docs/en/pages/guides/graphql/connected-users-dataset.mdx [hunk 6]
```diff
@@ -0,0 +1,175 @@
+---
+title: How to query Connected Users data with GraphQL API
+description: This guide will explain how to query Connected Used data using the GraphiQL playground.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/graphql/connected-users-dataset.mdx:3]:
```suggestion
description: This guide will explain how to query Connected Users data using the GraphiQL playground.
```

File: src/content/docs/en/pages/devtools/api-graphql/overview/overview.mdx (hunk 1 above)
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/api-graphql/overview/overview.mdx:38]:
```suggestion
- Consult the number of users connected to your live streamings.
```

File: src/content/docs/pt-br/pages/guias/graphql/connected-users-dataset.mdx [hunk 7]
```diff
@@ -0,0 +1,175 @@
+---
+title: Como consultar dados de usuários conectados com a GraphQL API
+description: Este guia explicará como consultar dados do dataset connectedUsers usando o playground GraphiQL.
+meta_tags: graphql, api, query, postman, live ingest, live streaming
+namespace: docs_guides_query_connected_users_graphql
+permalink: /documentacao/produtos/guias/query-dados-connected-users-com-graphql/
+menu_namespace: graphqlMenu
+---
+
+O conjunto de dados **connectedUsers** permite que você obtenha dados agregados em tempo real, fornecidos pelo Live Ingest da Azion, relacionados ao número de usuários atualmente conectados às suas aplicações com transmissões ao vivo. Este conjunto de dados faz parte da GraphQL API do Real-Time Metrics e é gerado a partir dos eventos HTTP brutos, que são populados apenas para clientes com configurações de transmissão ao vivo.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/graphql/connected-users-dataset.mdx:10]:
```suggestion
O conjunto de dados **connectedUsers** permite que você obtenha dados agregados em tempo real, fornecidos pelo Live Ingest da Azion, relacionados ao número de usuários conectados às suas aplicações com transmissões ao vivo. Este conjunto de dados faz parte da GraphQL API do Real-Time Metrics e é gerado a partir dos eventos HTTP brutos, que são populados apenas para clientes com configurações de transmissão ao vivo.
```

File: src/content/docs/pt-br/pages/devtools/api-graphql/visao-geral/visao-geral-gql.mdx [hunk 8]
```diff
@@ -35,6 +35,7 @@ Com a API GraphQL do Real-Time Metrics, você pode, por exemplo:
 - Buscar os IPs com mais requisições.
 - Ver os domínios com menos requisições.
 - Analisar a quantidade de requisições por método HTTP.
+- Consulte o número de usuários atualmente conectados às suas transmissões ao vivo.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/api-graphql/visao-geral/visao-geral-gql.mdx:38]:
```suggestion
- Consulte o número de usuários conectados às suas transmissões ao vivo.
```

File: src/content/docs/pt-br/pages/guias/graphql/connected-users-dataset.mdx [hunk 9]
```diff
@@ -0,0 +1,175 @@
...
+
+Onde:
+
+| Campo | Descrição |
+|----------|----------|
+| `limit` | Especifica o número máximo de resultados a serem retornados. Máximo do sistema: `10000`   |
+| `filter` | Define os critérios usados para filtrar os dados retornados pela consulta |
+| `tsRange` | Um subcampo de `filter`. Especifica um intervalo de tempo para filtrar os dados. Inclui os campos `begin` e `end` para definir os timestamps de início e fim. Formato: `"YYYY-MM-DDTHH:mm:ss"`; exemplo: `"2024-04-11T00:00:00"` |
+| `groupBy` | Especifica os campos pelos quais os resultados da consulta devem ser agrupados. Exemplo: `[ts]` |
+| `orderBy` | Especifica a ordem em que os resultados devem ser retornados. Valores aceitos: `[ts_DESC]`, para ordem decrescente, e `[ts_ASC]`, para ordem crescente |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/graphql/connected-users-dataset.mdx:139]:
```suggestion
| `orderBy` | Especifica a ordem em que os resultados devem ser retornados. Exemplo: `[ts_DESC]`, para ordem decrescente, e `[ts_ASC]`, para ordem crescente |
```

**Review by lfsigreja — APPROVED** [review]

**Pull request merged**